### PR TITLE
OOP

### PR DESCRIPTION
**Pull request by kovalov**
### OOP

[Demo](https://kovalov.github.io/a-tiny-JS-world/) |
[Code base](https://github.com/kovalov/a-tiny-JS-world)

The code is submitted in a dedicated feature branch.

Only code files are submitted.

Please, review.

### CODE REVIEW

File: submissions/kovalov/oop/index.js [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+    this._properties = [
+      'species',
+      'name',
+      'gender',
+      'legs',
+      'hands',
+      'saying',
+    ];
+  }
+  set properties(prop) {
```
Comment (artem-trubin) [inline submissions/kovalov/oop/index.js:18]: Is this `set` property used somewhere?

**Review by artem-trubin — COMMENTED** [review]
Good job! A clever use of `toString` and `String`.

File: submissions/kovalov/oop/index.js (hunk 1 above)
Comment (kovalov) [reply submissions/kovalov/oop/index.js:18]: Have added it to have ability to add some props in future and just wanted to practice setters/getters:)

**Comment by OleksiyRudenko** [comment]
@kovalov please mark items on [**Minimal requirements** list](https://github.com/kottans/frontend-2022-homeworks/pull/32#issuecomment-1199497814) you genuinely believe are properly addressed by the code.

**Optional level-up** and **Bonus** lists are optional.

**Pull request closed without merging**